rollup.config.js

```diff
@@ -2,18 +2,19 @@ import typescript from 'rollup-plugin-typescript2'
 import commonjs from '@rollup/plugin-commonjs'
 import nodeResolve from '@rollup/plugin-node-resolve'
 import replace from '@rollup/plugin-replace'
+// import analyze from 'rollup-plugin-analyzer'
```

Comment:
remove it?

Reply:
Leaving it can help to analyze bundles in the future for debug purposes easily. IMO no need to change